index.html

```diff
@@ -7,6 +7,12 @@
     <link rel="stylesheet" href="public/css/style.css">
     <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Yrsa'>
     <title>Norizon | Links</title>
+    <meta name="title" content="norizon.xyz">
+    <meta name="description" content="Description and stuff here">
+    <meta name="keywords" content="frontend, norizon, dev">
+    <meta name="theme-color" content="#353746">
+    <meta name="robots" content="noindex, follow">
```

Comment:
```suggestion
    <meta name="robots" content="index, follow">
``` w/e I'll change this soon if needed